src/main/java/com/epam/preprod/sirenko/entity/Food.java

```diff
@@ -0,0 +1,60 @@
+package com.epam.preprod.sirenko.entity;
+
+import java.util.Objects;
+
+/**
+ * Food entity.
+ *
+ * @author E.Sirenko
+ **/
+public class Food extends Products {
+	private int amount;
+	private String petType;
```

Comment:
Do we need 'petType' field of Food?What does it mean?

Reply:
refactored it to petGroup enum